staff/sys/makevm

```diff
@@ -201,9 +212,13 @@ def _main(args):
                         help='amount of disk storage (in GB)')
     parser.add_argument('-V', '--vg', type=str, default='vg',
                         help='LVM volume group to use for storage')
+    parser.add_argument('-C', '--ceph', action='store_true', default=False,
+                        help='use Ceph storage instead of LVM')
+    parser.add_argument('-P', '--ceph-pool', type=str, default='vm',
+                        help='Ceph Pool to use for storage')
     parser.add_argument('--os-type', type=str, default='linux',
                         help='os type')
-    parser.add_argument('--os-variant', type=str, default='debian9')
+    parser.add_argument('--os-variant', type=str, default='debian10')
```

Comment:
I thought that debian10 didn't exist in virt-install yet?

Reply:
`osinfo-query os | grep Debian` seems to indicate yes on pandemic but not riptide so maybe it's a buster thing?

Reply:
we probably shouldn't change the default yet in that case.